### ISSUE TITLE
React print: unknown HTML node tag "br"

### ISSUE THREAD

**Issue by usu** [ecamp/ecamp3#3571]
linebreak to be implemented in react HTML parser

**Comment by carlobeltrame** [comment]
Will be fixed in #3532

**Comment by BacLuc** [comment]
Now it does not crash, but it also doesn't make the line break:

[Stadtgame.pdf](https://github.com/ecamp/ecamp3/files/12447493/Stadtgame.pdf)

HTML:
```
<p>Alle Gruppen waren bei allen Posten. Nun müssen sie die Tickets zusammensetzten</p><p>The User is currently already redirected to the admin overview of a camp after creating it.<br />I thought this issue was about displaying a text after that happens, I would have used the Toast (since it's already in the project) for that. </p>
```

Browser:
![grafik](https://github.com/ecamp/ecamp3/assets/1506818/d1b6b2fa-8d8f-498d-af00-1d7dd379a252)

This happens if you paste html from another website or use shfit+enter
